Example/Auth/ApiTests/AuthCredentialsTemplate.h

```diff
@@ -62,7 +72,10 @@ The ID of the test user in the Custom Auth token.
 #define KGOOGLE_USER_NAME $KGOOGLE_USER_NAME
 #define KFACEBOOK_APP_ID $KFACEBOOK_APP_ID
 #define KFACEBOOK_APP_ACCESS_TOKEN $KFACEBOOK_APP_ACCESS_TOKEN
-#define KFACEBOOK_USER_NAME $KFACEBOOK_USER_NAME
-#define KCUSTOM_AUTH_TOKEN_URL $KCUSTOM_AUTH_TOKEN_URL
-#define KCUSTOM_AUTH_TOKEN_EXPIRED_URL $KCUSTOM_AUTH_TOKEN_EXPIRED_URL
-#define KCUSTOM_AUTH_USER_ID $KCUSTOM_AUTH_USER_ID
+#define KFACEBOOK_USER_NAME @"Fname Lname"
+
+// The custom auth test is currently disabled. These values are ignored (though
+// must be present).
+#define KCUSTOM_AUTH_TOKEN_URL @"Disabled test"
+#define KCUSTOM_AUTH_TOKEN_EXPIRED_URL @"Disabled test"
+#define KCUSTOM_AUTH_USER_ID @"Disabled test"
```

Comment:
I think it's better to have macros in the Template files or developers may think `KFACEBOOK_USER_NAME` has to be `Fname Lname`